lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -829,8 +838,12 @@ def format_idempotency_key(options, suffix)
 
       def success_from(response, options)
         if response['status'] == 'requires_action' && !options[:execute_threed]
```

Comment:
When inquiring on pending transaction it fails here because the request type doesn't have execute_threed. If you add `options[:request_type] != :inquire` it should fix it.